### ISSUE TITLE
Match ESLint and Prettier rules

### ISSUE THREAD

**Issue by NovAti0n** [obiwac/sinfbot#1]
At the moment, the two files are completely different and contain rules that are incompatible (i.e. Prettier states that there shouldn't be any semicolon while ESLint states the opposite). We should decide on what code style to follow.

**Comment by obiwac** [comment]
why do we have eslint and prettier at the same time in the first place lol

personally I'm for no semicolons, they add nothing

**Comment by obiwac** [comment]
i don't think it makes sense. eslint also acts as a formatter because it modifies code. best to have only one code-modifying "thing"
